runners/core-java/src/main/java/org/apache/beam/runners/core/construction/SerializablePipelineOptions.java

```diff
@@ -59,7 +59,6 @@ public PipelineOptions get() {
   private void readObject(ObjectInputStream is) throws IOException, ClassNotFoundException {
     is.defaultReadObject();
     this.options = deserializeFromJson(serializedPipelineOptions);
-    // TODO https://github.com/apache/beam/issues/18430: remove this call.
```

Comment:
Why is this no longer relevant? This still seems dicy (unconditionally overwriting the options as a side effect of deserialization).

Reply:
Restored TODO statement here for now